scripts/build.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/bash -l
+# only execute these lines if the `module` command is present in the environment
+# used for the BU SCC
+if command -v module &>/dev/null; then
+    module load gcc/12.2.0
+    module load miniconda
+    conda env create -f "/projectnb/respond/environment.yml" 2&>/dev/null
+    conda activate respond
+fi
+
+# help message to be output either with the -h flag or when using invalid syntax
+showhelp () {
+    printf "\033[31m%s\033[0m" "$0 - Build the DataManagement Model"
+    echo
+    echo
+    echo "Syntax: $(basename "$0") [-h|-t OPTION|-p|-l|-n]"
+    echo "h              Print this help screen."
+    echo "t OPTION       Set the build type to OPTION"
+    echo "               Options: [Debug|Release]"
+    echo "               Default: Debug"
+    echo "p              Build and run tests."
+    echo "l              Build Shared Library."
+    echo "n              Build Benchmarking executable"
```

Comment:
Maybe remove this flag? I'm assuming this came from copy + paste of RESPOND.

Reply:
Yeah, in theory we'd be able to build them completely separately and we have tests, build modes, library typing, and benchmarking but again, that is a problem for when we wanna mess with the build process.